### PR TITLE
DROOLS-6271 Drools phreak rete diagram

### PR DESCRIPTION
**Pull request by tarilabs**
**JIRA**: https://issues.redhat.com/browse/DROOLS-6271

**referenced Pull Requests**: 

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1624
* https://github.com/kiegroup/drools/pull/3530

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: drools-retediagram/src/test/resources/logback.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%date{HH:mm:ss.SSS} [%thread] %-5level %class{36}.%method:%line - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <logger name="org.kie" level="info"/>
+  <logger name="net.tarilabs" level="info"/>
+
```
Comment (tkobayas) [inline drools-retediagram/src/test/resources/logback.xml:28]: If you set this info level, we can easily check the generated file path. WDYT?
```
<logger name="org.drools.retediagram" level="info"/>
```

**Review by tkobayas — APPROVED** [review]
Great work, Matteo!

File: drools-retediagram/src/test/resources/logback.xml (hunk 1 above)
Comment (tarilabs) [reply drools-retediagram/src/test/resources/logback.xml:28]: ops :) I forgot to update this, nice catch !

Comment (tarilabs) [reply drools-retediagram/src/test/resources/logback.xml:28]: resolved with 47f74eb

**Review by hellowdan — APPROVED** [review]

**Review by mariofusco — APPROVED** [review]

**Pull request merged**